.env.example

```diff
@@ -0,0 +1,7 @@
+DELAY_OSM=
```

Comment:
Can this example file contain example values ? In particular for :
- Delay OSM
- OSM API URL (https://master.apis.dev.openstreetmap.org/)
- OSM lang : en

Reply:
Sure, what `DELAY_OSM` do you currently use on prod?

Reply:
I think it's 300000 (5 minutes in milliseconds)